packages/worker/src/debug-replayer/worker-thread.ts

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Not a big deal, but I'd rename this to workflow-debugging-worker-thread.ts or something else that clearly set it appart from workflow-worker-thread.ts.

Reply:
Don't mind changing this.